### DISCUSSION TITLE
I'm using `->rules()` in a form, but I want to differentiate the rules for the create page and the edit page.

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#9158] [Help] [answered]
In Filament, the form is the same for both create and edit, but I want to branch the rules inside `->rules()`. 
Is this possible?

**Comment by leandrocfe** [comment, marked as answer]
You can use the context/operation, for example 

`Field::make('your_field')->required(fn (string $operation): bool => $operation === 'create')`

**Reply by bzy107** [reply]
Thank you. The issue has been resolved.
